llm/patches/02-shutdown.diff

```diff
@@ -1,96 +0,0 @@
-diff --git a/examples/server/server.cpp b/examples/server/server.cpp
-index a0b46970..7800c6e7 100644
```

Comment:
This is in mainline now?

Reply:
Yes it is! https://github.com/ggerganov/llama.cpp/pull/5244